src/index.css

```diff
@@ -14,14 +15,167 @@ body {
     sans-serif;
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
+  margin: 0;
+  padding: 0;
 }
 
-code {
-  font-family:
-    source-code-pro,
-    Menlo,
-    Monaco,
-    Consolas,
-    'Courier New',
-    monospace;
+a {
+  text-decoration: none;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  background: rgba(165, 42, 89, 0.644);
+
+  /* background-color: #3a386c82; */
```

Comment:
- [ ] Kindly remove this commented code. One more step and we are good.

Reply:
oh sorry i really forgot, lemme do it asap